style.css

```diff
@@ -94,94 +113,372 @@ ul {
   list-style: none;
 }
 
-hr {
-  height: 1px;
-  background: #dfe1e6;
-}
-
 @media screen and (min-width: 340px) {
   #header {
+    display: flex;
+    flex-direction: row;
+    justify-content: space-between;
+    align-items: center;
     background-color: white;
     padding: 4px;
   }
 
-  .log {
-    height: 30px;
-    width: 30px;
+  .logo-place {
+    padding: 10px 12px;
+    margin-left: 12px;
+    color: var(--primaryColor);
+    font-style: normal;
+    font-weight: 700;
+    font-size: 18px;
+    line-height: 20px;
+  }
+
+  .Menu {
+    display: none;
+    gap: 1rem;
+    padding: 8px;
+    margin-right: 24px;
+  }
+
+  .m_item {
+    color: #344563;
+    padding: 0.5rem;
+  }
+
+  .Menuicon {
+    color: var(--primaryColor);
+    padding: 10px 12px;
+    margin-left: 12px;
     display: flex;
-    align-items: center;
   }
+}
+
+#headline_section {
+  background-color: white;
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  padding: 114px 24px;
+  gap: 12px;
+  background-image: url(./img/Headerbg.svg);
+  background-repeat: no-repeat;
+  background-size: cover;
+  border-radius: 0 0 0 5rem;
+}
 
-  .hamburger {
+.headline {
+  display: flex;
+  flex-direction: column;
+  align-items: flex-start;
+  gap: 2rem;
+  padding: 1rem;
+}
+
+#lts_connect {
+  color: var(--primaryColor);
+}
+
+.contact {
+  list-style: none;
+  display: flex;
+  gap: 1rem;
+}
+
+.icons {
+  width: 24px;
+  height: 24px;
+}
+
+#works {
+  display: grid;
+  align-content: center;
+  padding: 114px 24px;
+  grid-gap: 88px;
+}
+
+.pages {
+  display: grid;
+  align-content: center;
+  padding: 16px;
+  background: white;
+  border: 1px solid #def1e6;
+  border-radius: 16px;
+}
+
+.card_image {
+  padding-bottom: 1rem;
+}
+
+.detail1 {
+  display: grid;
+  align-content: start;
+  grid-gap: 12px;
+  width: 100%;
+}
+
+.title {
+  display: grid;
+  justify-content: start;
+}
+
+.history {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: center;
+  align-items: center;
+  grid-gap: 12px;
+}
+
+.details {
+  padding: 8px 0.2px;
+}
+
+.bouton {
+  display: grid;
+  grid-auto-flow: column;
+  justify-content: start;
+  align-items: center;
+  grid-gap: 8px;
+}
+
+.btn {
+  float: left;
+  padding-right: 0.5rem;
+}
+
+.btntext {
+  background: #ebebff;
+  border-radius: 8px;
+  padding: 4px 12px;
+  color: var(--primaryColor);
+}
+
+#about_myself {
+  display: grid;
+  padding: 114px 24px;
+  grid-gap: 24px;
+  background: white;
+  border-radius: 0 5rem 0 0;
+}
```

Comment:
- [ ] Please kindly ensure that the chevrons have uniform designs according to the given template. I observed that one is bold than the others. 👍 
- [ ] More so, try and adjust the horizontal bar as it were in the previous review. **If you can design it closely to the figma template, I think that would be commendable** ℹ️ 

<img width="863" alt="Untitled" src="https://github.com/medijdir/portfolio_project/assets/103386183/4ee8538d-b016-48c9-87bf-3198f18fb941">